distributed/scheduler.py

```diff
@@ -4563,6 +4815,15 @@ async def add_nanny(self, comm):
         }
         return msg
 
+    def get_task_duration(self, ts: TaskState) -> double:
+        return self.state.get_task_duration(ts)
+
+    def get_comm_cost(self, *args, **kwargs):
+        return self.state.get_comm_cost(*args, **kwargs)
+
+    def check_idle_saturated(self, *args, **kwargs):
+        return self.state.check_idle_saturated(*args, **kwargs)
```

Comment:
These all are only accessed by stealing. Why not let stealing access `scheduler.state` instead?